qipr_approver/approver/templates/approver/dashboard_su.html

```diff
@@ -0,0 +1,130 @@
+</div>
+<div class="container__dashboard">
+<div class="row dashboard_title-bg">
+        <div class="col s12 dashboard_title">
+                <h3 class="dashboard_title-main">QIPR Dashboard</h3>
+        </div>
+    </div>
+
+    <div class="row dashboard_border">
+        <div class="col s12 m2 dashboard_menu">
+            <ul>
+                <li class="dashboard_menu-item-active">Projects</li>
+                <li class="dashboard_menu-item"><a href="{% url 'approver:aboutyou' %}">Profile</a></li>
+            </ul>
+        </div>
+
+        <div class="col s12 m10 dashboard_border-left">
+            <h3 class="center-align dashboard_projects-title">QI Projects for {{ person.first_name }} {{ person.last_name }}</h3>
+            <div class="row">
+                <div class="col s6">
+                    <a class="dashboard_add" href="{% url 'approver:projects' %}"><div class="dashboard_add-btn">+</div><div class="dashboard_add-text">Add Project</div></a>
+                </div>
+                 <div class="col s6">
+                    <div class="input-field dashboard_search">
+                        <form method="post" action="{% url 'approver:dashboard_su' %}">
+                            {% csrf_token %}
+                            <input placeholder="Search your project title" class="dashboard_search-input" id="search" type="search" name="search" value={{ search_query }} >
+                            
+                            <input class="dashboard__search--projects" type="submit" value="&#128270;">
+                        </form>
+                    </div>
+                </div>
+            </div>
+
+
+            <table class="striped">
+                <thead>
+                    <tr>
+                        <th class="dashboard_row-title" data-field="title"><h5>Title</h5></th>
+                        <th class="dashboard_row-title" data-field="role"><h5>Role</h5></th>
+                        <th class="dashboard_row-title dashboard_row-title-project project__title--forms" data-field="Approval"><h5>QI Forms <a class="waves-effect waves-light dashboard__question modal-trigger" href="#modal1">?</a></h5>
+
```

Comment:
We should probably place another column with the PQI name so the admin knows whose project it is, or even maybe swap it out with the Role column since this is obsolete for the admin